modules/fun.js

```diff
@@ -216,7 +216,7 @@ async function slashFunMinesweeper(int) {
   // console.log(spaces);
   // seperate into rows and emojify and hide if not exposed
   const rowStrings = board.map(row => row.map(num => num < 0 ? mineSweeperEmojis[-num - 1] : `||${mineSweeperEmojis[Math.min(num, 9)]}||`).join(""));
-  if (!int.channel) {
+  if (!int.channel || !int.guild?.members.me?.permissionsIn(int.channel + "").has("SendMessages")) {
```

Comment:
For some reason the commands errored in the error channel, but worked everywhere else. I tried this to fix it but that didn't work. It could reply but it couldn't send in the channel... I don't know if it needs fixing but it is a bug seemingly...

Reply:
done